ansible/roles/installation/tasks/main.yml

```diff
@@ -44,6 +44,8 @@
     THREEMARB_API_IDENTITY: "{{ rails.threema.api_identity }}"
     THREEMARB_API_SECRET: "{{ rails.threema.api_secret }}"
     THREEMARB_PRIVATE: "{{ rails.threema.private_key }}"
+    SIGNAL_SERVER_PHONE_NUMBER: "{{ rails.signal.server_phone_number | default('') }}"
```

Comment:
If it has a `default` it shouldn't be marked as `REQUIRED` in the `ansible/generate_config.sh`, no?